xcode/Subconscious/Shared/Services/DatabaseService.swift

```diff
@@ -1003,15 +1032,19 @@ final class DatabaseService {
                 .queryFTS5(link.slug.description),
                 .text(link.id)
             ]
-        ).compactMap({ row in
-            let did = row.col(0)?.toString()?.toDid()
+        ).compactMap({ row -> EntryStub? in
+            guard let did = row.col(0)?.toString()?.toDid() else {
+                return nil
+            }
```

Comment:
The `did` column of the `memo_search` table is marked as nullable, unlike the `did` column of `memo`. I'm not sure why, so flagging it in-case I'm missing something.